model/flow/identifier_test.go

```diff
@@ -116,3 +118,19 @@ func referenceMerkleRoot(t *testing.T, ids ...flow.Identifier) flow.Identifier {
 	copy(root[:], hash)
 	return root
 }
+
+func TestCIDConversion(t *testing.T) {
```

Comment:
this is just converting a fixture back and forth right? might make sense just to add a few quick lines to explain